aidemo.py

```diff
@@ -232,11 +244,13 @@ def setup_layout(self):
         picture_box.pack_start(self.image_celeb, False, True, 0)
         picture_box.pack_start(self.top5_grid, False, True, 0)
 
-        content_box = Gtk.Box(spacing=20)
+        content_box = Gtk.Box(spacing=10)
         content_box.pack_start(stream_box, True, True, 0)
         content_box.pack_start(picture_box, True, True, 0)
         main_box = Gtk.Box(orientation=Gtk.Orientation.VERTICAL, spacing=10)
-        main_box.pack_start(self.main_label, True, True, 0)
+        if SCREEN == 'HDMI' | FULLSCREEN:
```

Comment:
```suggestion
        if SCREEN == 'HDMI' or FULLSCREEN:
```
`|` is a bitwise or which does not work with `str` and `bool`.
Alternatively you could use:
```suggestion
        if (SCREEN == 'HDMI') | FULLSCREEN:
```
Although the first one looks more appealing to me.

Reply:
Most appealing would be `if (SCREEN == 'HDMI' || FULLSCREEN) {...`